### PR TITLE
[B2BP-245] NextJS deploy configuration

### PR DESCRIPTION
**Pull request by simosalsi**
We need to deploy a static website to an S3 bucket so that it can be reached by end users.
To be able to generate such files NextJS has to be configured accordingly.

#### List of Changes

- Set build output to be a static export

#### Motivation and Context

Needed for deployment of static website.

#### How Has This Been Tested?

Tested locally using `npm run build`

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by maurodandrea** [comment]
I've added the environment var CDN_PATH in PR https://github.com/pagopa/b2b-portals/pull/56/ ,variable that must be created on github along with the other variables indicated in ticket https://pagopa.atlassian.net/browse/B2BP-225

**Review by maurodandrea — APPROVED** [review]

File: apps/nextjs-website/next.config.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {}
+const nextConfig = {
+    output: 'export',
+    assetPrefix: process.env.CDN_PATH,
```
[the remark this answers is not in the document]
Comment (simosalsi) [reply apps/nextjs-website/next.config.js:4]: Removing `assetPrefix` creates a broken build when ran locally, so it was added to at least give the option to quickly add the `CDN_PATH` environment variable to be able to make sure everything renders/loads correctly (instead of having to wait for a developer to push the update if need be).

I would personally want to deploy a test website before entirely removing the value.

We could simply build two test websites (one with `assetPrefix` and one without) and run it in an environment resembling our production environment.

[the remark this answers is not in the document]
Comment (simosalsi) [reply apps/nextjs-website/next.config.js:4]: Keeping `assetPrefix` also has the slight advantage of making testing builds locally that much smoother, while `CDN_PATH` should be able to just be left empty in production environments without repercussions.

[the remark this answers is not in the document]
Comment (datalek) [reply apps/nextjs-website/next.config.js:4]: I didn't know it would break; actually, I haven't tried to build it 😄

**Review by datalek — APPROVED** [review]
LGTM
